requirements.txt

```diff
@@ -55,3 +55,4 @@ uWSGI==2.0.19.1
 # watchtower is for Cloudwatch logging integration
 watchtower==0.8.0
 Werkzeug==1.0.1
+webpub-manifest-parser
```

Comment:
As for now I uploaded `webpub-manifest-parser` to my personal PyPi account, once it's merged I'll move it to NYPL's account